_posts/2020-11-17-quarkiverse.adoc

```diff
@@ -0,0 +1,95 @@
+
+The `quarkiverse-parent` artifact (inherited by each extension in Quarkiverse) enables plugins that will format your source code to the same format used in Quarkus core.
+
+Documentation aggregation::
+
+Quarkiverse Hub uses https://antora.org[Antora] to aggregate each extension documentation in the Quarkiverse docs website. Check it out at https://quarkiverse.github.io/quarkiverse-docs/
+
+Automated inclusion in the upcoming Extension registry::
+
+The Extension registry is still a work in progress (https://registry.quarkus.io), but the gist of it is to have a single place to browse for extensions
+(including https://code.quarkus.io, command line and IDE plugins).
+
+== Requirements
+
+In order to automate extension releases and publication of docs, there are some rules that need to be followed for projects under the Quarkiverse organization:
+
+- The extension repository should be named `quarkiverse-<project>`
+- A Quarkiverse extension MUST belong to the `io.quarkiverse.<project>` groupId
+- The root pom.xml MUST inherit from `io.quarkiverse:quarkiverse-parent`
+- A Quarkiverse extension has the following structure:
+
+[tree]
+----
+├── deployment <1>
+├── runtime
+├── integration-test
+├── docs <2>
+├── pom.xml
+├── LICENSE
+└── README
+----
+
+<1> The *deployment*, *runtime* and *integration-test* directories containing the extension classes. See link:https://quarkus.io/guides/writing-extensions#create-new-extension-modules-using-maven[How to create new extension modules using Maven] for more details.
+<2> *docs* (optional): The docs should follow the link:https://docs.antora.org/antora/2.3/standard-directories/[Antora's Standard File and Directory Set]. See an example link:https://github.com/quarkiverse/quarkiverse-freemarker/tree/b41701b05f5886fd7c91148cdd273028fcd28950/docs[here].
+
+For more information, look at the link:https://github.com/quarkiverse/quarkiverse/wiki[Quarkiverse Wiki]
+
+image:/assets/images/posts/quarkiverse/quarkiverse-way.jpg[]
+
+== How do I sign-up?
+
+Create an *Extension Proposal* ticket in the link:https://github.com/quarkusio/quarkus/issues[Quarkus issue tracker] (first check if the extension you're proposing is not created yet) and let us know if you want to lead the development of this extension by commenting on the issue.
+
+The Quarkus team will get you started with the boring stuff of setting up a brand new repository and deployment scripts, so you'll have fun coding it.
+
+As usual, we're always improving, so take a look at link:https://github.com/quarkiverse/quarkiverse/wiki#joining-quarkiverse[Joining Quarkiverse] for the process we are currently using.
+
+== What extensions are in Quarkiverse already?
+
+Check out the link:https://github.com/search?q=topic%3Aquarkus-extension+org%3Aquarkiverse&type=Repositories[extensions that are already there].
+
+Stay tuned for the latest info about Quarkiverse in the link:https://github.com/quarkiverse/quarkiverse/wiki[Quarkiverse Wiki]
```

Comment:
should we add anotice indicating that you can still have an project external to quarkus and quarkiverse - just that then there are more manual steps to do ?